plone/app/z3cform/templates/file_input.pt

```diff
@@ -0,0 +1,110 @@
+<div
+    class="form-group"
+    i18n:domain="plone"
+    tal:attributes="id view/id;
+                    style view/style;
+                    title view/title;
+                    lang view/lang;
+                    onclick view/onclick;
```

Comment:
I always disliked the z3c.form widget template style of including an incomplete and mostly obsolte list of attributes directly here.
Can't that be done by e.g. ``<div ${view/attributes_rendered} />``. Not sure if this would work.